integr8ly/integreatly-operator
# Mgdapi 113

# Description
Update keycloak-operator to set CPU and Memory limits

#Verification
install on fresh cluster :
```
make cluster/prepare/local INSTALLATION_TYPE=managed-api
make code/run INSTALLATION_TYPE=managed-api
```

Once complete:
- Verify that sso pods have cpu and memory limits set
- Verify that you can login to 3scale with an IDP user.
